lib/Models/Catalog/Ows/WebFeatureServiceCatalogItem.ts

```diff
@@ -235,6 +238,65 @@ class GetCapabilitiesStratum extends LoadableStratum(
       return keyword === "GEOSERVER";
     }
   }
+
+  // Check if geojson output is supported (by checking GetCapabilities OutputTypes OR FeatureType OutputTypes)
+  hasJsonOutputFormat = (outputFormats: string[] | undefined) => {
+    return isDefined(
+      outputFormats?.find((format) =>
+        ["json", "JSON", "application/json"].includes(format)
+      )
+    );
+  };
+
+  // Helper function for get outputFormat()
+  supportsGeojson =
+    this.hasJsonOutputFormat(this.capabilities.outputTypes) ||
+    [...this.capabilitiesFeatureTypes.values()].reduce<boolean>(
+      (hasGeojson, current) =>
+        hasGeojson && this.hasJsonOutputFormat(current?.OutputFormats),
+      true
+    );
```

Comment:
This is non-observable - which may cause issues as `this.capabilitiesFeatureTypes` is `@computed`.

To fix this you can put it inside `get outputFormat` or create a new `@computed` property

Reply:
I agree, I don't think we need a `supportsGeojson` trait.

It's totally fine to create `@computed` properties that aren't in the traits system.

Reply:
Ah I was thrown by this error when I had it marked as `observable`: 
```
Catalog item could not be added to workbench.: [mobx] Trying to declare a computed value for unspecified getter 'supportsGeojson'
  Failed to load `NDW Hydrants` metadata
```

Reply:
Done, put inside `get outputFormat`